### PR TITLE
Fix(readme): Add more documentation on getting started.

### PR DESCRIPTION
**Pull request by wirehead**
- Fixing the make target in the readme (misspelling)
- Adding link to Glide.
- Gopath, mercurial, and source location.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,10 +37,11 @@ Make sure you have the prerequisites:
 - Go 1.6
 - A running Kubernetes cluster
 - `kubectl` properly configured to talk to your cluster
-- Glide 0.10 or greater
+- [Glide](https://glide.sh/) 0.10 or greater with both git and mercurial installed.
```
Comment (technosophos) [inline README.md]: Git and Mercurial are not part of Glide's requirements -- they're part of Go's requirements.

**Comment by technosophos** [comment]
Thanks! When the CLA passes, I'll LGTM and merge this.

**Comment by wirehead** [comment]
I signed it!

**Comment by wirehead** [comment]
I signed it! (again)

**Comment by wirehead** [comment]
I signed it!

**Comment by technosophos** [comment]
@wirehead thanks for going through the CLA rigamarole. Merged.

**Pull request merged**